guides/common/modules/proc_enabling-the-installer-managed-dhcp-service.adoc

```diff
@@ -0,0 +1,53 @@
+[id="enabling-the-installer-managed-dhcp-service"]
+= Enabling the installer-managed DHCP service
+
+If you do not have a DHCP server available in your network, you can use the installer-managed DHCP service.
+This feature enables you to provide a DHCP service with a low maintenance effort.
```

Comment:
Something feels off about the phrasing. Do you think this would work better?
```suggestion
This feature enables you to provide a DHCP service with low maintenance overhead.
```